### DISCUSSION TITLE
502 nginx bad gateway when trying to upload student proff

### DISCUSSION THREAD

**Discussion by tamirhad** [community/community#21009] [GitHub Education]
when trying to apply for github education by uploading a proff of my student ID(through: https://education.github.com/discount_requests/7997919/additional_information),  im getting 502 Bad Gateway by nginx.

**Comment by jayded** [comment]
I believe I have this issue as well: [education.github.com](education.github.com) gives me a 502 bad gateway.

**Reply by hansnery** [reply]
Same problem here. Uploaded proof but the submission never completed. Tried refreshing the page and now I'm getting this error. Too bad because all my projects at Heroku will be lost.

**Reply by Jenieee3** [reply]
I got the same message

**Comment by talonendm** [comment]
I got the same message. However, I got confirmation by email to my University email and everything works now..

**Comment by thingizkhan** [comment]
same problem please someone help

**Reply by sskhati2** [reply]
Same here, no fix that I know of for this yet

**Reply by dksilv4** [reply]
Same here

**Comment by Bad3r** [comment]
GH support sent us here and no one is helping..

**Comment by ryauy** [comment]
same problem here

**Comment by x0b3** [comment]
same problem here 2

**Comment by x0180** [comment]
I have the same problem, **help**!!

**Comment by ryauy** [comment]
They just need time to process it. It'll take a few hours to a day, so there's no need to contact the support yet. Not sure why they give an Nginx error til then though

**Reply by TheTempest77** [reply]
Are you sure? Giving a 502 error is really misleading then, it could have just told us that our application is being processed. I guess I'll check back in a day or so and see if it's fixed. Thanks!

**Comment by TheTempest77** [comment]
Same problem here please help

**Comment by rureddish** [comment]
Same problem here!

**Comment by RH-98** [comment]
Same problem here!

**Comment by TheGreatDaniad** [comment]
same here!! 
any fix?

**Reply by JonathanLorand** [reply]
same problem too

**Comment by ejxousiva** [comment]
same problem here!

**Comment by Ception** [comment]
Same here.

**Comment by bagusrizkis** [comment]
Same here

**Comment by arshadalisoomro** [comment]
Same Problem here! I am confused, what is wrong with it, where is GitHub team? Why they are not fixing it Man!!

**Comment by itsamrit** [comment]
Same problem here 502 bad gateway nginx error.
